libs/sdk-backend-base/src/decoratedBackend/securitySettings.ts

```diff
@@ -0,0 +1,28 @@
+// (C) 2021 GoodData Corporation
+
+import { ISecuritySettingsService, ValidationContext } from "@gooddata/sdk-backend-spi";
+import identity from "lodash/identity";
+
+/**
+ * @alpha
+ */
+export type UrlValidationResultWrapper = (result: boolean) => boolean;
+
+/**
+ * @alpha
+ */
+export abstract class DecoratedSecuritySettingsService implements ISecuritySettingsService {
+    public scope: string;
+
+    protected constructor(
+        private decorated: ISecuritySettingsService,
+        protected readonly wrapper: UrlValidationResultWrapper = identity, // TODO PR Question is this necessary? What is the purpose of this optional wrapper?
```

Comment:
you don't need this wrapper here. the reason why it exists on some other decorators like exec factory is to cater for case where you want to keep the original behavior of the factory BUT want to decorate the instances which the factory creates.

Reply:
The wrapper was removed.